packages/flutter_tools/lib/src/dart/analysis.dart

```diff
@@ -46,7 +46,7 @@ class AnalysisDriver {
   DriverOptions options;
 
   String get sdkDir {
-    return options.dartSdkPath ?? fs.path.absolute(cli_util.getSdkDir().path);
+    return options.dartSdkPath ?? fs.path.absolute(cli_util.getSdkPath());
```

Comment:
Agh - that's what I get for not syncing! We're still missing the pubspec declaration. I'll rebase and update the comment.